packages/grid/x-data-grid/src/utils/domUtils.ts

```diff
@@ -49,3 +49,15 @@ export const getActiveElement = (root: Document | ShadowRoot = document): Elemen
 
   return activeEl;
 };
+
+export function isEventTargetInPortal(event: React.SyntheticEvent) {
```

Comment:
I extracted this to a util function and used it everywhere where we handle portals.